src/features/content-explorer/content-explorer/ContentExplorerSelectAll.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import Checkbox from '../../../components/checkbox/Checkbox';
+
+import messages from '../messages';
+
+class ContentExplorerSelectAll extends PureComponent {
```

Comment:
Since this component doesn't contain any state variables, can we convert it to a functional component?